### PR TITLE
BL-4684: Delete sub when mot expiry date > 70 months old

### PR DESCRIPTION
**Pull request by raffcecco**

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/motr/notifier/processing/service/SubscriptionHandlerHelper.java [hunk 1]
```diff
@@ -8,6 +10,8 @@
  */
 public class SubscriptionHandlerHelper {
 
+    public static final int monthsBeforeDeletion = 70;
```
Comment (kevinc-kainos) [inline src/main/java/uk/gov/dvsa/motr/notifier/processing/service/SubscriptionHandlerHelper.java]: maybe 'private'
I think the convention was to capitalise consts to be something like MONTHS_BEFORE_DELETION
(although an exception I've noticed is logger)

Comment (aaronlong) [inline src/main/java/uk/gov/dvsa/motr/notifier/processing/service/SubscriptionHandlerHelper.java]: Should we have this passed in as a config value?

Comment (aaronlong) [inline src/main/java/uk/gov/dvsa/motr/notifier/processing/service/SubscriptionHandlerHelper.java]: The formatting for statics should be 'MONTHS_BEFORE_DELETION' :)

File: src/main/java/uk/gov/dvsa/motr/notifier/processing/service/ProcessSubscriptionService.java [hunk 2]
```diff
@@ -23,12 +24,11 @@
 import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.motDueDateUpdateRequired;
 import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.motTestNumberUpdateRequired;
 import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.oneMonthEmailRequired;
+import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.subscriptionDeletionRequired;
 import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.twoWeekEmailRequired;
 import static uk.gov.dvsa.motr.notifier.processing.service.SubscriptionHandlerHelper.vrmUpdateRequired;
 
-
 public class ProcessSubscriptionService {
-
```
Comment (aaronlong) [inline src/main/java/uk/gov/dvsa/motr/notifier/processing/service/ProcessSubscriptionService.java]: Needs a line break :)

File: src/integration-test/java/uk/gov/dvsa/motr/test/integration/unloader/SubscriptionDbItemQueueItemUnloaderTest.java [hunk 3]
```diff
@@ -73,7 +77,7 @@ public void cleanUp() {
         fixture.removeItem(subscriptionItem);
     }
 
-    @Test
+    // @Test
     public void whenAnItemIsInTheDb_TheLoaderAddsToQueue_ThenTheNotifierSuccessfullyProcessesIt()
```
Comment (Thomas-Franklin) [inline src/integration-test/java/uk/gov/dvsa/motr/test/integration/unloader/SubscriptionDbItemQueueItemUnloaderTest.java]: Okay, there appears to be several tests here commented out

Comment (raffcecco) [inline src/integration-test/java/uk/gov/dvsa/motr/test/integration/unloader/SubscriptionDbItemQueueItemUnloaderTest.java]: Yea, sorry - commented those out to speed up testing something. Back in now.

**Pull request merged**